### PR TITLE
adding validation for subscription

### LINKED ISSUE #235: [Design QA] Newsletter sign up - add error message if box is unchecked
If someone clicks subscribe but does not check the "I'm okay with Mozilla handling my info as explained in this Privacy Notice" box then they should get an error message saying to check it.

### PR DESCRIPTION
**Pull request by gvn**
closes #235

### CODE REVIEW

**Comment by gvn** [comment]
Did this as an alert for now after discussing w. Kristina.

File: app/scss/main.scss [hunk 1]
```diff
@@ -113,3 +113,7 @@ body {
   flex-direction: column;
   justify-content: center;
 }
+
+.inline-block {
```
Comment (alanmoo) [inline app/scss/main.scss]: giphy-1

File: app/react/components/footer/footer.jsx [hunk 2]
```diff
@@ -14,16 +30,19 @@ export default React.createClass({
...
             </div>
             <div className="row">
               <div className="col-sm-12 checkbox">
                 <label>
-                  <input type="checkbox"/> Im okay with Mozilla handling my info as explained in this <a href="#">Privacy Notice</a>
+                  <input ref="inputOK" type="checkbox"/> Im okay with Mozilla handling my info as explained in this <a href="#">Privacy Notice</a>
                 </label>
               </div>
+              <div className="col-sm-12">
+                <div hidden={!this.state.hasAttemptedToSubmit || this.state.hasConsent} className="inline-block alert alert-danger m-t-2">Please check the box above if you want to subscribe.</div>
```
Comment (alanmoo) [inline app/react/components/footer/footer.jsx]: Haven't run code yet, but should this be `&&` ?

File: app/scss/main.scss (hunk 1 above)
Comment (gvn) [inline app/scss/main.scss]: No? I couldn't find a class in Bootstrap that does this FWIW.

Comment (alanmoo) [inline app/scss/main.scss]: Yeah, you probably could have made it a span and given the wrapper a margin class for the same effect, but it's not a big deal.

**Pull request merged**